python/ray/rllib/optimizers/async_replay_optimizer.py

```diff
@@ -154,9 +154,7 @@ def step(self):
                         info["td_error"])
                     if "stats" in info:
                         self.stats[pid] = info["stats"]
-            # send `replay` back also so that it gets released by the original
-            # thread: https://github.com/ray-project/ray/issues/2610
-            self.outqueue.put((ra, replay, prio_dict, replay.count))
+            self.outqueue.put((ra, prio_dict, replay.count))
```

Comment:
What is `replay` and why is it removed now? Does this have to do with the crash?